src/main/java/com/wanderersoftherift/wotr/core/rift/RiftLevelManager.java

```diff
@@ -46,7 +46,11 @@
 
 public class RiftLevelManager {
```

Comment:
Should this also be final and private constructor?

Reply:
I would say so.